openpype/hosts/resolve/hooks/post_last_workfile.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+from openpype.lib import PostLaunchHook
+from openpype.lib.execute import run_subprocess
+
+
+SCRIPT = r"""
+from openpype.pipeline import install_host, registered_host
+import DaVinciResolveScript as bmd
+
+def main():
+    import openpype.hosts.resolve.api as bmdvr
+
+    # activate resolve from openpype
+    install_host(bmdvr)
+
+    # Open last workfile
+    workfile_path = r"{workfile_path}"
+    if workfile_path:
+        host = registered_host()
+        host.open_file(workfile_path)
+
+    # Launch the OpenPype menu inside the host
+    menu_script_path = r"{openpype_menu_path}"
+    print(f"Running script {{menu_script_path}}")
+    resolve = bmd.scriptapp("Resolve")
+    fusion = resolve.Fusion()
+    fusion.RunScript(menu_script_path)
+
+if __name__ == "__main__":
+    tried = 0
+    resolve = None
+    while tried < {try_count}:
+        print("Waiting for Resolve to start ...")
+        try:
+            resolve = bmd.scriptapp("Resolve")
+            main()
+            break
+        except:
+            pass
+
+        tried += 1
```

Comment:
As far as I know you don't need to `retry` yourself since you can just pass a `timeout` argument to the `scriptapp` function and set it to e.g. 30 seconds so it'll just try continuously during the 30 seconds automatically.

Reply:
The timeout didn't affect the real timeout at all on our side.